controllers/gslb_controller.go

```diff
@@ -238,12 +242,12 @@ func (r *GslbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 		if strategy == failoverStrategy {
 			for annotationKey, annotationValue := range a.Meta.GetAnnotations() {
-				if annotationKey == "k8gb.io/primarygeotag" {
+				if annotationKey == primaryGeoTagAnnotation {
 					gslb.Spec.Strategy.PrimaryGeoTag = annotationValue
 				}
 			}
 			if gslb.Spec.Strategy.PrimaryGeoTag == "" {
-				log.Info("k8gb.io/primarygeotag annotation is missing, skipping Gslb creation...")
+				log.Info(fmt.Sprintf("%s annotation is missing, skipping Gslb creation...", primaryGeoTagAnnotation))
```

Comment:
Isn't it better to use Infof() here?

Reply:
Makes sense, @k0da, but we also should maintain consistency, and rest of the code uses `Info(fmt.Sprintf())`, not `Infof()`.
If you're OK, I can make the proposed replacement (35 changes in 5 files) as separate PR.

Reply:
@somaritane, good point for sure! If you decide to reactor, please can you check log.Error, log.Debug ,etc... We can do all at once.

Reply:
Maybe we can do in separate PR.

Reply:
@somaritane ack

Reply:
Also, after carefully looking, this `log` is of `logr.Logger` type, [logr](https://pkg.go.dev/github.com/go-logr/logr) package  doesn't have functions like `Infof()` or `Errorf()`, instead you should use `Info()` and `Error()` with structured logging format e.g.:
```diff
--- log.Errorf("unable to reconcile object %s/%s: %v", object.Namespace, object.Name, err)
+++ log.Error(err, "unable to reconcile object", "object", object)
```